src/common/hook/useQueue/utils/publish.tsx

```diff
@@ -97,7 +97,33 @@ export const getRawDocuments = async (forms: FormEntry[], config: Config): Promi
 };
 
 const wrapDocuments = async (rawDocuments: any[]) => {
-  return utils.isRawV3Document(rawDocuments[0]) ? await wrapDocumentsV3(rawDocuments) : wrapDocumentV2(rawDocuments);
+  return utils.isRawV3Document(rawDocuments[0]) ? await wrapDocumentsV3(rawDocuments) : wrapDocumentsV2(rawDocuments);
+};
+
+const processVerifiableDocument = async (
+  jobs: PublishingJob[],
+  nonce: number,
+  contractAddress: string,
+  verifiableDocuments: RawDocument[]
+) => {
+  if (verifiableDocuments.length === 0) return;
+  console.log(verifiableDocuments);
+  const rawDocuments = verifiableDocuments.map((doc) => doc.rawDocument);
```

Comment:
why do u do this again?

Reply:
inside rawDocument, there is other rawDocumentProperties. That give the correct OpenAttestation Raw Document

Reply:
let me rename to rawOpenAttestationDocument, will it be a bit clearer ? it is weird way of how the thing is structured